Readme.md

```diff
@@ -51,17 +50,17 @@ Supported units and abbreviations are as follows and are case-insensitive:
 | 1024<sup>7</sup> | ZiB  | zebibite  |
 | 1024<sup>8</sup> | YiB  | yobibite  |
 
+### Compatibility Binary (JEDEC)
 
-### Compatibility Binary Prefixes (JEDEC)
-
-Overwrites the lower units of the metric system with the commonly misused prefixes
+Overwrites the lower units of the metric system with the commonly misused values. Units greater than terabyte are not supported.
+This resembles popular but incorrect units used by e.g. the Windows OS and [bytes][bytes].
```

Comment:
I mean they're not "wrong", JEDEC was a standard at one point, "Outdated" is better?

OSX only converted after 10.15 afaik, which is relatively recent.  Unix itself finalized the conversions pretty recently.